### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by sanfordstudent**

### CODE REVIEW

File: course_discovery/apps/core/api_client/lms.py [hunk 1]
```diff
@@ -87,3 +87,46 @@ def get_api_access_request(self, user):
...
+        Get details for a course.
+
+        Arguments:
+            course_key (string): LMS identifier for the course.
+
+        Returns:
+            Body consists of the following fields:
+            * effort: A textual description of the weekly hours of effort expected
+                in the course.
+            * end: Date the course ends, in ISO 8601 notation
```
Comment (efischer19) [inline course_discovery/apps/core/api_client/lms.py:102]: [8601 4lyfe](https://xkcd.com/1179/)

File: course_discovery/apps/publisher/tests/utils.py [hunk 2]
```diff
@@ -1,7 +1,29 @@
+import datetime
+
+import mock
+from django.test import TestCase
+
 from course_discovery.apps.core.tests.factories import USER_PASSWORD, UserFactory
 from course_discovery.apps.publisher.tests import factories
 
 
+class MockedStartEndDateTestCase(TestCase):
```
Comment (efischer19) [inline course_discovery/apps/publisher/tests/utils.py:10]: This is a great idea for handling all these test cases

**Review by efischer19 — APPROVED** [review]
LGTM

File: course_discovery/apps/core/api_client/lms.py (hunk 1 above)
Comment (sanfordstudent) [reply course_discovery/apps/core/api_client/lms.py:102]: 🔥

**Pull request merged**